Gemfile.lock

```diff
@@ -57,15 +60,15 @@ GEM
     faraday-httpclient (1.0.1)
     faraday-multipart (1.0.4)
       multipart-post (~> 2)
-    faraday-net_http (1.0.1)
+    faraday-net_http (1.0.2)
     faraday-net_http_persistent (1.2.0)
     faraday-patron (1.0.0)
     faraday-rack (1.0.0)
     faraday-retry (1.0.3)
     faraday_middleware (1.2.0)
       faraday (~> 1.0)
-    fastimage (2.2.7)
-    fastlane (2.217.0)
+    fastimage (2.3.1)
+    fastlane (2.218.0)
```

Comment:
This is the part I don't get. When I run `bundle exec fastlane -v`, it tells me I'm running 2.218.0 (which matches this) but then it also says 2.221.1 is available and to run `bundle update fastlane`. But then I run that again and it continues to stay on 2.218.0. 🤷

Reply:
🤔 I updated ruby to 3.2.2 and was able to `bundle update fastlane` to 2.221.1. Should I push the changes and see what happens on your machine?

Reply:
I also put `gem 'fastlane', '~> 2.221'` in the Gemfile.

Reply:
Yeah, you can go ahead and push. Thanks

Reply:
The weird thing is that I'm on ruby 3.3.4. Who knows -- doesn't seem worth it to me to try to debug.

Reply:
oh 🤔 , 3.2.2 was the latest version listed in `rbenv` which is what I use to install ruby versions.

Reply:
I pushed and added a .ruby-version file which is picked up by some ruby tools and it seems good to have around.